.github/workflows/bats_test.yaml

```diff
@@ -6,7 +6,7 @@ on:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: true no i
```

Comment:
This does not seem a correct change.  Please remove or add details.

Reply:
Agreed this is very strange